src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import {
+  getAttachments,
+  createAttachment,
+  deleteAttachment
+} from "./attachments.resource";
+import Gallery from "react-grid-gallery";
+import styles from "./attachments-overview.css";
+
+export default function AttachmentsOverview() {
+  const [attachments, setAttachments] = useState([]);
+
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+
+  useEffect(() => {
+    if (patientUuid) {
+      const abortController = new AbortController();
+      getAttachments(patientUuid, true, abortController).then(
+        (response: any) => {
+          const listItems = response.data.results.map(attachment => ({
+            src: `/openmrs/ws/rest/v1/attachment/${attachment.uuid}/bytes`,
+            thumbnail: `/openmrs/ws/rest/v1/attachment/${attachment.uuid}/bytes`,
```

Comment:
Right now I'm using the source of the original image as the `src` value and `thumbnail` value. 
When [ATT-38 AttachmentBytesResource_* to take the (complex obs) view as parameter](https://issues.openmrs.org/browse/ATT-38) is complete, I should be able to access the thumbnail bytes.